### ISSUE TITLE
Forced Evaluations

### ISSUE THREAD

**Issue by nated099** [OpenPipe/openpipe#702]
Hello!

I'd like to start by saying I love this project so far it has been very helpful in some of my R&D on fine-tuning for some upcoming projects.

This is a feature request, or rather a feature _disable_ request option. Perhaps there is one already and I'm missing it, but I have looked.

There is alot of manual work in my process, even if synthetic data is used I must manually curate a good portion of it to ensure that training iterations are going as planned.

My current issue is the Evaluations. While this appears to be inspired towards automating FT model selection, it is forced and wastes some of my work. Since I am required to manually evaluate more complex training, the evaluations as they are are undesired for this specific task, I would rather train on the full dataset, then run tests against the FT after with a different dataset. 

- Manually prepared dataset of 3k-15k complex entries performing entity extraction, sentiment analysis + syntax formatting. 
- Since the expected completion is already in place with the dataset, these entries being used for evaluation is actually just a waste.
- 20% loss is a big hit for this, as I already stated I manually prepare the data and evaluating results afterwards due to strict requirements.

Per your docs:
```
When you train a model 20% of the dataset entries you provide will be withheld from training. These entries form your test set. For each entry in the test set, your new model will produce an output that will be shown in the [evaluation table](https://app.openpipe.ai/p/BRZFEx50Pf/datasets/0aa75f72-3fe5-4294-a94e-94c9236befa6/evaluate).
```

TLDR
I like the evaluation feature for various purposes, but would also like the option to disable it when fine-tuning. 

Thank you

**Comment by corbt** [comment]
Thanks for the report. You can get around this by manually setting `"split": "TRAIN"` in the jsonl file for all entries at the time you upload your dataset. More information in the docs here: https://docs.openpipe.ai/features/uploading-data

Let me know if that doesn't address the issue!

**Comment by nated099** [comment]
Fantastic, this resolved it. Thank you!

**Comment by nated099** [comment]
Fantastic, this resolved it. Thank you!
